src/components/TransactionsList.js

```diff
@@ -28,7 +29,7 @@ const TransactionsList = () => {
           </th>
         </tr>
 
-        {"... your code here..."}
+        {props.transactions.map(transaction => <Transaction transaction={transaction} searchTerm={props.searchTerm}/>)}
```

Comment:
Probably better to do the filtering here - only pass down the list of transactions that match, instead of making the `Transaction` component decide whether or not it should render.